src/Bridges/Variable/bridges/RSOCtoPSDBridge.jl

```diff
@@ -56,6 +56,8 @@ end
 const RSOCtoPSD{T,OT<:MOI.ModelLike} =
     SingleBridgeOptimizer{RSOCtoPSDBridge{T},OT}
 
+MOI.Bridges.bridging_cost(::Type{<:RSOCtoPSDBridge}) = 1.5
```

Comment:
So there's an argument to be made here. Would it be better to globally penalise PSD sets more than RSOC sets? We're starting to muck with bridge costs, and it becomes a bit subjective.